src/DataDepsGenerators.jl

```diff
@@ -84,38 +92,39 @@ function message(meta)
     register(DataDep(
         \"$(meta.shortname)\",
         \"\"\""""
-    netString = format_meta(meta.website, netString, "Website")
-    netString = format_meta(format_authors(meta.author), netString, "Author", true)
-    netString = format_meta(meta.maintainer, netString, "Maintainer", true)
-    netString = format_meta(meta.license, netString, "License", true)
-    netString = format_meta(meta.publishedDate, netString, "Date of Publication", true)
-    netString = format_meta(meta.createDate, netString, "Date of Creation", true)
-    netString = format_meta(meta.modifiedDate, netString, "Date of Modification", true)
-    netString = format_meta(meta.description, netString, missing,  true)
-    netString = format_meta(meta.paperCite, netString, "Please cite this paper", true)
-    netString = format_meta(meta.datasetCite, netString, "Please cite this dataset", true)
-    netString = strip(netString * "\n\"\"\",")
-    netString = format_meta(meta.dataurls, netString)
-    netString = netString * ","
-    netString = format_meta(format_checksums(meta.datachecksums), netString)
-    netString = strip(netString * "\n))")
+    netString *= format_meta(meta.website, "Website")
+    netString *= format_meta(meta.author, "Author")
```

Comment:
see this should be:
`format_meta(format_authors(meta.authors), "Author")`

Reply:
Then I have to change the metadata struct type. I have used format_authors when instantiating a new metadata in `find_metadata()`.

Reply:
yes indeed.